src/frontend/src/handler/create_table.rs

```diff
@@ -217,13 +220,11 @@ pub(crate) fn gen_create_table_plan(
         properties,
         Info::TableSource(TableSourceInfo {}),
     )?;
-    let (plan, table) = gen_materialized_source_plan(context, source.clone(), session.user_id())?;
+    let (plan, table) = gen_materialize_plan(context, source.clone(), session.user_id())?;
```

Comment:
>     // TODO(Yuanxin): Detect if there is an external source based on `properties` (WITH CONNECTOR) and make prost source accordingly.

And if i understand correctly this source, the `source` parameter will be optional?

Reply:
Yes ✅